### PR TITLE
chore: Fix the row count index for stocky entry

### PR DESCRIPTION
**Pull request by shruti2323**
The count of row id was failing , had fix the issue.
Asana Link : https://app.asana.com/0/1200073273462988/1199966800575045

Had rebased the current feature branch (created from bs-jul-sprint-1)with old "feature_stockEntry" branch which (was created from bs-jun-sprint-2)

### CODE REVIEW

File: erpnext/stock/doctype/stock_entry/stock_entry.py [hunk 1]
```diff
@@ -690,6 +691,21 @@ def update_stock_ledger(self):
 
 		self.make_sl_entries(sl_entries, self.amended_from and 'Yes' or 'No')
 
+	def validate_mandatory_batch_number(self):
+		"""
+			validate the stock items batch number is present or not
+		"""
+		items_without_batch_no = []
+
+		for idx, item in enumerate(self.get("items"), start=1):
```
Comment (shruti2323) [inline erpnext/stock/doctype/stock_entry/stock_entry.py:700]: with enumeration, the index 0 gives the first element, ideally which is correct, but we want to display the error msg starting from count= 1, not 0. Thus have added start=1, which starts count with the value 1 on the first loop iteration.
Refer below image:-
image

File: erpnext/controllers/queries.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from erpnext.stock.get_item_details import _get_item_tax_template
 from frappe.utils import unique
 
- # searches for active employees
+# searches for active employees
```
Comment (hrwX) [inline erpnext/controllers/queries.py:13]: Create this PR on `bs-jun-sprint-2` branch

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/controllers/queries.py (hunk 2 above)
Comment (sahil28297) [reply erpnext/controllers/queries.py:13]: @shruti2323 the `bs-jun-sprint-2` branch is already released now, if the code is not present in `bs-jul-sprint-1` branch, then createt a new PR on `pre-release` branch.

Comment (shruti2323) [reply erpnext/controllers/queries.py:13]: This PR is already rebased.

**Comment by sahil28297** [comment]
@shruti2323 refer https://github.com/Bloomstack/erpnext/pull/1442. Closing this

**Pull request closed without merging**